tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -116,6 +119,60 @@ def verify_email(self, email):
         """Handle user email exceptions."""
         pass
 
+    def _handle_user(self, email, sub, auth_options):
+        """Handle user."""
+        User = get_user_model()
+
+        # Check if a user with the same email already exists
+        user = User.objects.filter(email=email).first()
+
+        if user and auth_options.get("login_gov_uuid", False):
+            # Setup log context
+            logger_context = {
+                "user_id": user.id,
+                "content_type": SecurityEventToken,
+                "object_id": user.id,
```

Comment:
since the `content_type` is `SecurityEventToken`, i think your `object_id` needs to be a `SecurityEventToken` object as well (or `content_type` has to be `User`). currently nothing shows in the object column in DAC. SET and user both make sense to me